themes/theme-b2b/tailwind.config.ts

```diff
@@ -3,6 +3,7 @@ import type { Config } from 'tailwindcss';
 import { preset } from './src/preset';
 
 export default {
+  plugins: [require('tailwindcss-react-aria-components')],
```

Comment:
If we add it to our `createPreset` helper every theme will get it! :)

Reply:
now they are seperated